### ISSUE TITLE
Visiting storybook produces a console error on visit

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#2095]
```
[mobx] Encountered an uncaught exception that was thrown by a reaction or observer component, in: 'Reaction[Autorun@191]' TypeError: Cannot convert undefined or null to object
    at keys (<anonymous>)
    at Function.keys (es5-shim.js:1148)
    at Object.get allRefNames (assembly.ts:185)
    at trackDerivedFunction (mobx.module.js:761)
    at ComputedValue.push.../../node_modules/mobx/lib/mobx.module.js.ComputedValue.computeValue (mobx.module.js:1258)
    at ComputedValue.push.../../node_modules/mobx/lib/mobx.module.js.ComputedValue.trackAndCompute (mobx.module.js:1243)
    at ComputedValue.push.../../node_modules/mobx/lib/mobx.module.js.ComputedValue.get (mobx.module.js:1203)
    at shouldCompute (mobx.module.js:681)
    at ComputedValue.push.../../node_modules/mobx/lib/mobx.module.js.ComputedValue.get (mobx.module.js:1202)
    at shouldCompute (mobx.module.js:681)
    at Reaction.push.../../node_modules/mobx/lib/mobx.module.js.Reaction.runReaction (mobx.module.js:1758)
    at runReactionsHelper (mobx.module.js:1898)
    at reactionScheduler (mobx.module.js:1876)
    at mobx.module.js:1905
    at batchedUpdates$1 (react-dom.development.js:22380)
    at reactionScheduler (mobx.module.js:1905)
    at runReactions (mobx.module.js:1881)
    at endBatch (mobx.module.js:1581)
    at _endAction (mobx.module.js:967)
    at executeAction (mobx.module.js:921)
    at bound setLoaded (mobx.module.js:901)
    at runMiddleWares (mobx-state-tree.module.js:2563)
    at runWithActionContext (mobx-state-tree.module.js:2426)
    at res (mobx-state-tree.module.js:2453)
    at _callee$ (index.ts:695)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:293)
    at Generator.next (runtime.js:118)
    at asyncGeneratorStep (asyncToGenerator.js:3)
    at _next (asyncToGenerator.js:25)
```

This does not happen in latest release, v1.3.0 storybook, so it is just main branch that has issue now

**Comment by cmdcolin** [comment]
Fixed by 502c82f

Tricky one since it was a weird typo that I got to create this in the first place, and no typescript error since it is using "generic object" so .keys was assumed to exist

**Comment by cmdcolin** [comment]
could point to a need to test embedded code more e.g. with integration tests
